manifests/params.pp

```diff
@@ -39,15 +39,23 @@
           $plugin_config_dir           = '/etc/collectd/managed_config'
           $collectd_config_file        = '/etc/collectd/collectd.conf'
           $filtering_config_file       = '/etc/collectd/filtering_config/filtering.conf'
-          $mysql_socket_file           = '/var/lib/mysqld/mysqld.sock'
+          $bundled_plugins             = [
+                                            'collectd-apache',
+                                            'collectd-kafka',
+                                            'collectd-memcached',
+                                            'collectd-mysql',
+                                            'collectd-nginx',
+                                            'collectd-postgresql',
+                                            'collectd-varnish'
+                                         ]
```

Comment:
travisci failed:
manifests/params.pp - ERROR: two-space soft tabs not used on line 50